include/afl-fuzz.h

```diff
@@ -149,6 +149,17 @@ struct tainted {
 
 };
 
+typedef struct {
+  long data_type;
+  int  data_num;
+  char data_buff[1024];
+} My_message;
+
+typedef struct {
+  long data_type;
+  int  data_num[2];
+} Rw_message;
```

Comment:
Please use more clear naming instead of `My_message` and `Rw_message`. Also, it is advised that you follow the naming convention of the project. For this one, use `some_type_t` would be better.